cron/getIncrementBalance.js

```diff
@@ -0,0 +1,46 @@
+const cron = require('node-cron');
+const { familyModel, userModel, transactionModel } = require('../database/models');
+
+const { getLogger } = require('../helpers');
+const logger = getLogger('getIncrementBalance');
+
+async function main() {
+  try {
+    await cron.schedule('0 0 1 * *', async function () {
+
+      const allFamilies = await familyModel.find({});
+
+      await Promise.all(allFamilies.map(async item => {
+        await item.update(
+          {
+            $inc: {
+              giftsForUnpacking: Math.floor((item.balance * item.flatSquareMeters / item.flatPrice) - item.giftsUnpacked)
+            }
+          }
+        )
+
+        const user = await userModel.findOne({ familyId: item.familyId });
+        const sum = item.totalSalary + item.passiveIncome;
+
+        // awaiting transactionModal realization 
+        const itemBalance = await transactionModel.mounthlyAccrual(sum, item.incomePercentageToSavings, user._id, item._id);
```

Comment:
You could do this method by your own. Crons and everything related to it - it's your feature